.github/workflows/_e2e_tests.yaml

```diff
@@ -1,17 +1,28 @@
 name: e2e tests
-run-name: e2e tests, branch:${{ github.ref_name }}, triggered by @${{ github.actor }}
-
-concurrency:
-  # Limit the concurrency of e2e tests to run only 1 workflow for ref (branch).
-  # Ref: https://docs.github.com/en/actions/using-jobs/using-concurrency
-  group: ${{ github.workflow }}-${{ github.ref }}
-  cancel-in-progress: true
```

Comment:
Don't we want the concurrency of these jobs to be limited? i.e. 1 workflow run per branch?

Reply:
We do and that's preserved - we're still limiting the concurrency on the level of the top-most workflow which is `e2e_targeted.yaml`.